health-services/individual/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 All notable changes to this module will be documented in this file.
 
+
+## 1.1.7 - 2025-04-29 
+
+ - Added validators for beneficiary ID validation in individual creation and update flows.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)